src/httpclient.c

```diff
@@ -2,15 +2,23 @@
 
 #include <corto/httpclient/httpclient.h>
 #include <curl/curl.h>
+
 #define INITIAL_BODY_BUFFER_SIZE (512)
 
+/* Logging Support */
 corto_buffer g_logBuffer;
 bool g_logSet = false;
-
 struct url_data {
     size_t size;
     char* buffer;
 };
+
+/* Local Configuration */
+static corto_threadKey TLS_ConnectTimeoutKey;
+static corto_threadKey TLS_TimeoutKey;
```

Comment:
It is preferable to have a single key for all configuration items, as this list will likely grow in the future (this will require you to allocate memory and free it up in a destructor function).

Reply:
Naming convention for keys thus far has been something like:
```
corto_threadKey PACKAGE_KEY_CONFIGURATION_ITEM
```
Suggest to rename to something like
```
corto_threadKey HTTPCLIENT_KEY_CONFIG
```